scripts/scanner.lua

```diff
@@ -988,16 +1018,8 @@ end
 --Examines a forest position and classifies it by tree density. Used for the scanner list.
 function mod.classify_forest(position,pindex,drawing)
    local tree_count = 0
-   local tree_group = game.get_player(pindex).surface.find_entities_filtered{type = "tree", position = position, radius = 16, limit = 15}
-   if drawing then
-      rendering.draw_circle{color = {0, 1, 0.25},radius = 16,width = 4,target = position, surface = game.get_player(pindex).surface, time_to_live = 60, draw_on_ground = true}
-   end
-   for i,tree in ipairs(tree_group) do
-      tree_count = tree_count + 1
-      if drawing then
-         rendering.draw_circle{color = {0, 1, 0.5},radius = 1,width = 4,target = tree.position, surface = tree.surface, time_to_live = 60, draw_on_ground = true}
-      end
-   end
+   local tree_count  = game.get_player(pindex).surface.count_entities_filtered{type = "tree", position = position, radius = 16, limit = 15}
```

Comment:
We don't currently have a proper way to enable debugging.  The only thing we could do here is refactor so that you can uncomment some call to another function.

Also the problem is I don't even know what this is trying to show or for how long, just that it issues thousands of draw calls.  What does it allow one to debug exactly?  It seems to me that for example if you have multiple forests at the same time you don't actually get to know which one goes with which call, just that hey, there were lots of trees for some reason.

should we consider moving forward and having you or @EphDoering write something in renders.lua that renders a list of entities in whatever way you're trying to be able to see here generically if some debug flag is enabled?  I think that is a better approach because in the bigger picture no blind dev is going to be able to keep these working reliably long-term.  When I go after rewriting this properly, admittedly low on my list, the new version will only have such if not-me adds it.  Which isn't a statement on value, it's a statement of capability.  I'm not questioning this in general, only that this specific one specifically seems not useful and pointing out that I'm blind and many others who might contribute are so we can't propose better solutions.

Now all that said if you do actually find this one useful specifically I can go do the commented-function-call thing or something.